generic_tracers/generic_COBALT.F90

```diff
@@ -496,456 +508,507 @@ subroutine user_add_params
+    call get_param(param_file, "generic_COBALT", "ki_mdz",           zoo(2)%ki,                "ki_mdz",           units="mol N kg-1", default=1.25e-6)                        ! moles N kg-1
+    call get_param(param_file, "generic_COBALT", "ki_lgz",           zoo(3)%ki,                "ki_lgz",           units="mol N kg-1", default=1.25e-6)                        ! moles N kg-1
+    call get_param(param_file, "generic_COBALT", "ktemp_smz",        zoo(1)%ktemp,             "ktemp_smz",        units="C-1", default=0.063)                   ! C-1
+    call get_param(param_file, "generic_COBALT", "ktemp_mdz",        zoo(2)%ktemp,             "ktemp_mdz",        units="C-1", default=0.063)                   ! C-1
+    call get_param(param_file, "generic_COBALT", "ktemp_lgz",        zoo(3)%ktemp,             "ktemp_lgz",        units="C-1", default=0.063)                   ! C-1
+    call get_param(param_file, "generic_COBALT", "irr_mem_dpthresh1",cobalt%irr_mem_dpthresh1, "irr_mem_dpthresh1",units="watts m-2", default=30.0) ! watts m-2
+    call get_param(param_file, "generic_COBALT", "irr_mem_dpthresh2",cobalt%irr_mem_dpthresh2, "irr_mem_dpthresh2",units="watts m-2", default=10.0) ! watts m-2
+    call get_param(param_file, "generic_COBALT", "dpause_max",       cobalt%dpause_max,        "dpause_max",       units="", default=0.0)           ! dimensionless
+    call get_param(param_file, "generic_COBALT", "upswim_chl_thresh",zoo(1)%upswim_chl_thresh, "upswim_chl_thresh",units="", default=0.0) ! dimensionless
+    call get_param(param_file, "generic_COBALT", "upswim_chl_thresh",zoo(2)%upswim_chl_thresh, "upswim_chl_thresh",units="", default=0.0) ! dimensionless
+    call get_param(param_file, "generic_COBALT", "upswim_chl_thresh",zoo(3)%upswim_chl_thresh, "upswim_chl_thresh",units="", default=0.0) ! dimensionless
+    call get_param(param_file, "generic_COBALT", "upswim_I_thresh",  zoo(1)%upswim_I_thresh,   "upswim_I_thresh",  units="", default=0.0)     ! dimensionless
+    call get_param(param_file, "generic_COBALT", "upswim_I_thresh",  zoo(2)%upswim_I_thresh,   "upswim_I_thresh",  units="", default=0.0)     ! dimensionless
+    call get_param(param_file, "generic_COBALT", "upswim_I_thresh",  zoo(3)%upswim_I_thresh,   "upswim_I_thresh",  units="", default=0.0)     ! dimensionless
+    call get_param(param_file, "generic_COBALT", "swim_max",         zoo(1)%swim_max,          "swim_max",         units="m day-1", default=100.0)
+      zoo(1)%swim_max = zoo(1)%swim_max/sperd! max swimming (m sec-1)
+    call get_param(param_file, "generic_COBALT", "swim_max",         zoo(2)%swim_max,          "swim_max",         units="m day-1", default=500.0)
+      zoo(2)%swim_max = zoo(2)%swim_max/sperd! max swimming (m sec-1)
+    call get_param(param_file, "generic_COBALT", "swim_max",         zoo(3)%swim_max,          "swim_max",         units="m day-1", default=2000.0)
+      zoo(3)%swim_max = zoo(3)%swim_max/sperd! max swimming (m sec-1)
```

Comment:
Since we have 3 zooplankton groups here,  I suggested to change these codes as follows:"
```
    call get_param(param_file, "generic_COBALT", "upswim_chl_thresh1",zoo(1)%upswim_chl_thresh, "upswim_chl_thresh1",units="", default=0.0) ! dimensionless
    call get_param(param_file, "generic_COBALT", "upswim_chl_thresh2",zoo(2)%upswim_chl_thresh, "upswim_chl_thresh2",units="", default=0.0) ! dimensionless
    call get_param(param_file, "generic_COBALT", "upswim_chl_thresh3",zoo(3)%upswim_chl_thresh, "upswim_chl_thresh3",units="", default=0.0) ! dimensionless
    call get_param(param_file, "generic_COBALT", "upswim_I_thresh1",  zoo(1)%upswim_I_thresh,   "upswim_I_thresh1",  units="", default=0.0)     ! dimensionless
    call get_param(param_file, "generic_COBALT", "upswim_I_thresh2",  zoo(2)%upswim_I_thresh,   "upswim_I_thresh2",  units="", default=0.0)     ! dimensionless
    call get_param(param_file, "generic_COBALT", "upswim_I_thresh3",  zoo(3)%upswim_I_thresh,   "upswim_I_thresh3",  units="", default=0.0)     ! dimensionless
    call get_param(param_file, "generic_COBALT", "swim_max1",         zoo(1)%swim_max,          "swim_max1",         units="m day-1", default=100.0)
      zoo(1)%swim_max = zoo(1)%swim_max/sperd! max swimming (m sec-1)
    call get_param(param_file, "generic_COBALT", "swim_max2",         zoo(2)%swim_max,          "swim_max2",         units="m day-1", default=500.0)
      zoo(2)%swim_max = zoo(2)%swim_max/sperd! max swimming (m sec-1)
    call get_param(param_file, "generic_COBALT", "swim_max3",         zoo(3)%swim_max,          "swim_max3",         units="m day-1", default=2000.0)
      zoo(3)%swim_max = zoo(3)%swim_max/sperd! max swimming (m sec-1)
```